features/javascript/validate.feature

```diff
@@ -0,0 +1,159 @@
+@stable @javascript
+Feature: validate
+
+  Scenario: HTTP Request
+    Given you expect the following HTTP Request:
+    """
+    {
+      "method": "POST",
+      "headers": {
+        "user-agent": "curl/7.24.0 (x86_64-apple-darwin12.0) libcurl/7.24.0 OpenSSL/0.9.8x zlib/1.2.5",
+        "host": "httpbin.org",
+        "accept": "*/*"
+      },
+      "body": ""
+    }
+    """
+    And the actual HTTP Request equals:
+    """
+    {
+      "method": "POST",
+      "uri": "/ip",
+      "headers": {
+        "user-agent": "curl/7.24.0 (x86_64-apple-darwin12.0) libcurl/7.24.0 OpenSSL/0.9.8x zlib/1.2.5",
+        "host": "httpbin.org",
+        "accept": "*/*"
+      },
+      "body": ""
+    }
+    """
+    When Gavel validates the HTTP message
+    # When you call:
+    # """
+    # gavel.validate(expectedRequest, actualRequest)
+    # """
```

Comment:
Feels like in this case it should be the step?

```gherkin
When I call gavel.validate(expected, actual)
```

To keep it DRY, you can easily map both "When you call gavel.validate(expected, actual)" and "When Gavel validates the HTTP message" to the same handling function so they're equivalent, but I wouldn't shy away from writing the scenario with the clearest words possible in the context.

Reply:
Note: it is a common practice to prefer "I" over "you" in case there's a person involved in the scenario. It helps with the empathy when reading.

Reply:
Thanks, I've added a new step definition and mapped it to the existing one.